recommend/runtimePolicy.go

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package recommend
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	opb "github.com/accuknox/auto-policy-discovery/src/protobuf/v1/observability"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const serviceAccData = "/run/secrets/kubernetes.io/serviceaccount"
+
+// CreateRuntimePolicy function generates runtime policy for service account
+func CreateRuntimePolicy(o *Options, img *ImageInfo) error {
```

Comment:
1. The first argument `(o *Options)` in this function is unused.
2. Since this function has become a one-liner [ `return summarySearch(img)` ], we could delete `summarySearch()` and handle its operation as part of `CreateRuntimePolicy()`.